src/Pim/index.tsx

```diff
@@ -107,7 +107,7 @@ type CollectionRoutesPropsType = RouteComponentProps<{}> & {
   componentEdit: any;
   componentView: any;
   items: {[key: string]: PimType};
-  onItemSave: (item: PimType, journalUid: string, originalContact?: PimType) => void;
+  onItemSave: (item: PimType, journalUid: string, originalItem?: PimType) => Promise<any>;
```

Comment:
Should be Promise<void> not any.

Reply:
`Promise<void>`?

Reply:
Sorry, that's what I meant, yeah. I forgot to wrap it in code tags so the `<void>` part was stripped. It seems.

Reply:
Ah yes, my favorite HTML tag: `<void>`

Reply:
hehe. :laughing: